README.md

```diff
@@ -61,6 +61,20 @@ directory for detailed copyright information.
 
 This project is licensed under the [Apache License v2.0][apache_license].
 
+Code of Conduct
+---------------
+
+CppMicroServices.org welcomes developers with different backgrounds and a broad range of 
+experience. A diverse and inclusive community will create more great ideas, provide more unique 
+perspectives, and produce more outstanding code. Our aim is to make the CppMicroServices community 
+welcoming to everyone.
+
+To provide clarity of what is expected of our members, CppMicroServices has adopted the code 
+of conduct defined by [contributor-covenant.org][contributor-covenant.org]. This document is used
+across many open source communities, and we believe it articulates our values well. 
+
+The full text is on [Code of Conduct](./CodeOfConduct.md).
```

Comment:
This sentence sounds awkward to me. See suggestions from my previous comment regarding this.